Hymns/Tags/TagStore.swift

```diff
@@ -4,9 +4,8 @@ import RealmSwift
 import Resolver
 
 protocol TagStore {
-    func storeFavorite(_ entity: FavoriteEntity)
+    func storeRealmObject<T: Object>(_ entity: T)
```

Comment:
What does this do? does making this function generic make both Favorites and Tags the same table?

Reply:
https://docs.swift.org/swift-book/LanguageGuide/Generics.html
Generics just allow you to not have to tie the function to one type in the parameters. So it could be called with any type as long as it conforms to the Object protocol here.